### ISSUE TITLE
PIR

### ISSUE THREAD

**Issue by michesmith** [bcgov/SIMS#2148]

**Comment by HRAGANBC** [comment]
Agreed, this is not ideal in terms of providing clarity to the student, however it is not critical, as the relevant information is provided on the Notice of Assessment, and is therefore viewable by the student before they approve their assessment and proceed. 

I do think that this should be targeted for an enhancement, and should be considered along with other forms of 'supporting information' which are not made very accessible to student or ministry staff after being provided. 
For example, when parents or partners provide supporting information, that information is not viewable by ministry staff when they're reviewing a student's application.

**Comment by michesmith** [comment]
@HRAGANBC  Is there some content we can add somewhere in the meantime,? If so what and where?

or 

should supporting information be added outside of the solution somewhere and we icebox this issue for now? 

Thank you
